tirgei/Questioner-API
# #163206275 Remove comments from code

#### What does this PR do?
This Pull Request removes comments from code

#### Description of Task to be completed?
The tasks for this PR is:
- Remove comments and update Docstrings to show what functions does

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner-API.git ```

CD into the project folder on your machine
```$ cd Questioner-API ```

Check out the branch for this PR
``` $ git checkout ch-remove-comments-163206275```

#### Pivotal Tracker story associated with this PR
[#163206275](https://www.pivotaltracker.com/story/show/163206275)
